Symfony/CS/Fixer/Symfony/NoEmptyCommentFixer.php

```diff
@@ -125,4 +133,41 @@ private function clearCommentToken(Tokens $tokens, $index)
         // override with whitespace token linebreak
         $tokens->overrideAt($index, array(T_WHITESPACE, "\n", $tokens[$index]->getLine()));
     }
+
+    private function isSurroundedBySingleLineComments(Tokens $tokens, $index)
+    {
+        $line = $tokens[$index]->getLine();
+        $prev = $tokens->getPrevNonWhitespace($index);
```

Comment:
please test previous (or next) line first and early return if possible, than check the other line if still needed